packages/markdown-docx/src/CiceroMarkToOOXMLTransformer.js

```diff
@@ -14,23 +14,73 @@
 
 'use strict';
 
+const { TEXT_RULE } = require('./rules');
+const {wrapOOXML} = require('./helpers');
+
+const definedNodes = {
+    computedVariable: 'org.accordproject.ciceromark.ComputedVariable',
+    heading: 'org.accordproject.commonmark.Heading',
+    item: 'org.accordproject.commonmark.Item',
+    list: 'org.accordproject.commonmark.List',
+    listBlock: 'org.accordproject.ciceromark.ListBlock',
+    paragraph: 'org.accordproject.commonmark.Paragraph',
+    softbreak: 'org.accordproject.commonmark.Softbreak',
+    text: 'org.accordproject.commonmark.Text',
+    variable: 'org.accordproject.ciceromark.Variable',
+    emphasize: 'org.accordproject.commonmark.Emph',
+};
+
+let globalOOXML='';
+
 /**
  * Transforms the ciceromark to OOXML
  */
 class CiceroMarkToOOXMLTransfomer {
 
-    definedNodes = {
-        computedVariable: 'org.accordproject.ciceromark.ComputedVariable',
-        heading: 'org.accordproject.commonmark.Heading',
-        item: 'org.accordproject.commonmark.Item',
-        list: 'org.accordproject.commonmark.List',
-        listBlock: 'org.accordproject.ciceromark.ListBlock',
-        paragraph: 'org.accordproject.commonmark.Paragraph',
-        softbreak: 'org.accordproject.commonmark.Softbreak',
-        text: 'org.accordproject.commonmark.Text',
-        variable: 'org.accordproject.ciceromark.Variable',
-        emphasize: 'org.accordproject.commonmark.Emph',
-    };
+    /**
+     * Gets the class of a given CiceroMark Node.
+     *
+     * @param {object} node CiceroMark node entity
+     * @returns {string} Class of given node
+     */
+    getClass(node) {
+        return node.$class;
+    }
+
+    /**
+     * Gets the OOXML for the given node.
+     *
+     * @param {object} node    Description of node type
+     * @param {object} counter Counter for different variables based on node name
+     * @param {object} parent  Parent object for a node
+     * @returns {string} OOXML for the given node
+     */
+    getNodes(node, counter, parent = null) {
+        if (this.getClass(node) === definedNodes.text) {
+            if (parent !== null && parent.class === definedNodes.emphasize) {
+                return TEXT_RULE(node.text, true);
```

Comment:
I would suggest you to create different rules for text and emphasized text. The reason is readability and let me elaborate over this.

What is more readable and quickly understandable?
```js
if (parent !== null &amp;&amp; parent.class === definedNodes.emphasize) {
  return TEXT_RULE(node.text, true);
} else {
  return TEXT_RULE(node.text);
}
```

```js
if (parent !== null &amp;&amp; parent.class === definedNodes.emphasize) {
  return EMPHASIS_RULE(node.text);
} else {
  return TEXT_RULE(node.text);
}
```

I really hope your answer is latter. The point of readability is not just making it easy to read but to understand as welll and that's why naming is important.